doc/specs/#2325 - Default Profile Settings.md

```diff
@@ -0,0 +1,314 @@
+  "defaults.json". It's different, but is that obvious?
+* `defaultProfileSettings`: Implies "settings of the default profile"
+* `defaults`: This kinda seems to conflict conceptually with "defaults.json"
+* `baseProfileSettings`: not the worst, but not terribly intuitive
+* Others considered with less enthusiasm
+    - `profiles.defaults`: people don't love the idea of a `.`, but hey, VsCode does it.
+    - `inheritedSettings`
+    - `rootSettings`
+    - `globalSettings`: again maybe conflicts a bit with other concepts/properties
+    - `profileSettings`
+    - `profilePrototype`
+
+##### Why is there this random floating profile in the global settings?
+
+Users may be confused about the purpose of this random `Profile` that's in the
+globals. What's that profile doing there? Is _it_ the default profile?
+
+### Proposal 2: `__default__` Profile object in the user's profiles
+```json
+{
+    "$schema": "https://aka.ms/terminal-profiles-schema",
+    "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+    "requestedTheme" : "dark",
+    "showTabsInTitlebar" : true,
+    "profiles":
+    [
+        {
+            "guid": "__default__",
+            "useAcrylic": true,
+            "acrylicOpacity": 0.1,
+            "fontFace": "Cascadia Code",
+            "fontSize": 10
+        },
+        {
+            "guid": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
+            "name": "Windows PowerShell",
+            "commandline": "powershell.exe",
+            "hidden": false
+        },
+        {
+            "guid": "{0caa0dad-35be-5f56-a8ff-afceeeaa6101}",
+            "name": "cmd",
+            "commandline": "cmd.exe",
+            "hidden": false
+        }
+    ],
+    "schemes": [],
+    "keybindings": []
+}
+```
+
+#### Benefits
+##### Encapsulates the default profile settings
+Puts all the default profiles settings in one object. Probably not as clear as
+proposal 1, since it could be _anywhere_ in the list of profiles.
+
+##### Groups default profile settings with profiles
+In this proposal, the default profile is grouped into the same list of objects
+as the other profiles. All the profiles, and the defaults are all under the
+`"profiles"` object. Makes sense.
+
+#### Concerns
+##### Mysterious `__defaults__` GUID
+The only way to _definitively_ identify that this profile is special is by
+giving it a constant string. This string is _not_ a guid, which again, would be
+obvious.
+
+##### Unintuitive
+Adding a profile that has a mysterious `guid` value use that profile as the
+"defaults" is _very_ unintuitive. Nothing aside from documentation would
+indicate to the user "hey, add this magic profile blob to use as defaults across
+all your profiles".
+
+##### Why does this one profile object apply to all the others
+It might be unintuitive that one profile from the list of profiles affects all
+the others.
+
+### Proposal 3: Change `profiles` to an object with a `list` of profiles and a `defaults` object

```

Comment:
Personal favorite (though I really just want inheritance). I really like the following aspects of it...
1. Easy schema change
2. Supporting the `"profiles": [ ... ]` vs `"profiles": { "defaults": { ... }, "list": [ ... ] }` syntax change is clever. I like the idea of just overloading it. Then _maybe_ deprecating support for the legacy version at some point. 
3. Scoping this to "profiles" is really easy to understand. No doubt that I am specifically making changes to my profiles

I'm thinking about what this would look like with the Settings UI. It would feel pretty intuitive. You could choose which specific profile you want to edit. Or just choose "edit my default profile settings" and it would edit this thing directly. 

Love it.